controllers/front/CartController.php

```diff
@@ -614,10 +627,9 @@ protected function areProductsAvailable()
             $currentProduct->hydrate($product);
 
             if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
-                return $this->trans(
-                   'The item %product% in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
-                    ['%product%' => $product['name']],
-                    'Shop.Notifications.Error'
+                return $this->translateProductNameWithAttributes(
+                   'The product [1]%product%[/1] in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
```

Comment:
```suggestion
                   'The product [1]%product%[/1] in your cart is now a product with combinations. Please delete it and choose one of its attributes to proceed with your order.',
```

Reply:
I'd change in:

> The product [1]%product%[/1] in your cart is now a product with combinations. Please delete it and choose one of its combination to proceed with your order.

Talking about attributes to a customer could be confusing, wdyt?

@Julievrz

Reply:
I'm not sure. Combinations ≠ attributes. 

A combination is a set of attributes that create different variations of a product for the customer to choose from. 

But I agree with you that if it's clear for merchants, it's not necessarily clear for their customers. 

Actually, I don't think the customer needs to know the details of the error here. It's confusing. The only thing they really need to do is choose a combination. 

So what about something like: 

"The product %product% is a product with combinations. Please choose one to proceed with your order." ?

Reply:
Done 😊